.github/workflows/lk2nd.yml

```diff
@@ -1,5 +1,5 @@
 name: lk2nd
-on: [push, pull_request]
+on: [workflow_dispatch, push, pull_request]
```

Comment:
Why would you need to modify this file?

Reply:
I want to remove those non-relevant files but I don't know how, I modified that because I needed it to compile

Reply:
I want to remove those non-relevant files but I don't know how, I modified that because I needed it to compile

Reply:
I already removed the file I didn't want thanks

Reply:
Well I don't think it's necessary to touch it, including removing it.